src/main/kotlin/no/nav/familie/ef/iverksett/brev/DistribuerVedtaksbrevTask.kt

```diff
@@ -74,7 +75,18 @@ class DistribuerVedtaksbrevTask(
         journalpostResultat: JournalpostResultat,
         behandlingId: UUID
     ) {
-        val bestillingId = journalpostClient.distribuerBrev(journalpostResultat.journalpostId, Distribusjonstype.VEDTAK)
+        val bestillingId = try {
+            journalpostClient.distribuerBrev(journalpostResultat.journalpostId, Distribusjonstype.VEDTAK)
+        } catch (e: RessursException) {
+            val cause = e.cause
+            if (cause is HttpClientErrorException.Conflict) {
+                logger.warn("Conflict: distribuering av brev allerede utført for journalpost: ${journalpostResultat.journalpostId}")
+                objectMapper.readValue<DistribuerJournalpostResponseTo>(e.ressurs.data.toString(), DistribuerJournalpostResponseTo::class.java).bestillingsId
```

Comment:
Småpirk, men med import av `import com.fasterxml.jackson.module.kotlin.readValue` (som vi bruker ellers) så kan man endre denne raden
```suggestion
                objectMapper.readValue<DistribuerJournalpostResponseTo>(e.ressurs.data.toString()).bestillingsId
```